examples/examples_nodejs_test.go

```diff
@@ -449,6 +449,19 @@ func TestAccKmsAliasTs(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestRegress1423Ts(t *testing.T) {
+	test := getJSBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir:           filepath.Join(getCwd(t), "regress-1423"),
+			RunUpdateTest: false,
+		})
+	test.ExpectRefreshChanges = false
```

Comment:
Noting that because AWS default tests options set these settings to skip features of ProgramTest, issues like 1423 do not get detected. Consider removing the SkipRefresh/ExpectRefreshChanges customizations from default ProgramTest options and only enabling them with a comment where truly needed.